test/mocha/core/curlify.js

```diff
@@ -3,247 +3,335 @@ import Im from "immutable"
 import curl from "core/curlify"
 import win from "core/window"
 
-describe("curlify", function() {
-
-    it("prints a curl statement with custom content-type", function() {
-        var req = {
-            url: "http://example.com",
-            method: "POST",
-            body: {
-                id: 0,
-                name: "doggie",
-                status: "available"
-            },
-            headers: {
-                Accept: "application/json",
-                "content-type": "application/json"
-            }
+describe("curlify", function () {
+  context("POST", function () {
+    it("prints a curl statement with custom content-type", function () {
+      const req = {
```

Comment:
You're using `const` instead of `let` in this file. The convention of this project is to use and prefer `let` instead of `const`. I don't agree with this convention, but nevertheless it's a convention and unless we change this convention globally we should adhere to it. Consistency is one of the most important aspect of code and without adhering to conventions we can never have consistency.